docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -104,19 +104,15 @@ The query-tee returns the `Content-Type` header, HTTP status code, and body of t
 The preferred backend can be configured via `-backend.preferred=<hostname>`.
 The value of the preferred backend configuration option must be the hostname of one of the configured backends.
 
-When a preferred backend is configured, the query-tee uses the following algorithm to select the backend response to send back to the client:
-
-1. If the preferred backend response status code is 2xx or 4xx, the query-tee selects the response from the preferred backend.
-1. If at least one backend response status code is 2xx or 4xx, the query-tee selects the first received response whose status code is 2xx or 4xx.
-1. If no backend response status code is 2xx or 4xx, the query-tee selects the first received response regardless of the status code.
+When a preferred backend is configured, the query-tee always returns the response from the preferred backend.
```

Comment:
```suggestion
When a preferred backend is configured, the query-tee always returns a response from the preferred backend.
```

Reply:
I think "returns the response" is more technically correct here.

Reply:
Agree with Arve - there is only one response from the preferred backend, so "returns the response" makes more sense here.